### PR TITLE
Replace Alef character with SVG

### LINKED ISSUE #176: Replace the "ALEF" utf8 symbol with a SVG
For consistency sake across OSes. As of today, the symbol is rendered in very different ways.

Note: this component could later be shared in a "common" UI library.

### PR DESCRIPTION
**Pull request by nop33**
Closes #176

### CODE REVIEW

File: src/components/AlefSymbol.tsx
Comment (lf94) [inline src/components/AlefSymbol.tsx:29]: Maybe to save you and future us some pain we can use `useTheme` here to get the color? Then the symbol can color itself.

**Review by lf94 — COMMENTED** [review]
Other than my one comment your code is great :+1: 

I still need to do a physical/visual test.

File: src/components/AlefSymbol.tsx
Comment (nop33) [reply src/components/AlefSymbol.tsx:29]: I don't see how using this hook can help me customize the color of the symbol based on the color of the text where it is used at. I need to set it to red when used in error messages, gray when used in input field labels, and green when used in confirmed transaction details.

File: src/components/Inputs/index.tsx
Comment (lf94) [inline src/components/Inputs/index.tsx:23]: Mmm, `placeholder` not already optional on `HTMLInputElement`? Interesting they did that.

Comment (nop33) [reply src/components/Inputs/index.tsx:23]: I need to override it because it expects `string | undefined` and it conflicts with the `null` or `ReactNode`.

**Comment by lf94** [comment]
@nop33 I'm quite busy actually and I trust what you've done, I'll approve and merge.

**Review by lf94 — APPROVED** [review]

**Comment by mvaivre** [comment]
Mhh, I was actually reviewing it, and testing "visually".
When we're not in a hurry, let's wait for all reviews before merging. Or at least ask for confirmation before merging. 
Thanks!

**Pull request merged**
